### ISSUE TITLE
Multi-Episode Filenames not working

### ISSUE THREAD

**Issue by ulrick65** [SickChill/SickChill#6008]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: 1f124b31b16303a511c86717f11dd28b9cbf7b46
OS: Ubuntu 18.04
Browser: Chrome
What you did: Rescan Files
What happened: Error, unabled to detect episode number
What you expected: Episode detected and files scanned.
Logs:
```
2020-02-22 16:51:54 INFO SHOWQUEUE-REFRESH :: 78896: No episode number found in /mnt/DS1512-Write/TV/Red vs Blue/Season 09/Red vs. Blue - S09E01E02E03E04E05E06E07E08E09E10E11E12E13E14E15E16E17E18E19E20 - Rounding Error, The Twins, Number One, Evacuation Plan, Realignment, Familiar Feelings, Case File 01.045.mkv, ignoring it
2020-02-22 16:51:54 INFO SHOWQUEUE-REFRESH :: 78896: parse_result: Red vs. Blue - S09 [ABD: False] [ANIME: False] [whichReg: [u'season_only', u'season_only']]
2020-02-22 16:51:54 DEBUG SHOWQUEUE-REFRESH :: Parsed /mnt/DS1512-Write/TV/Red vs Blue/Season 09/Red vs. Blue - S09E01E02E03E04E05E06E07E08E09E10E11E12E13E14E15E16E17E18E19E20 - Rounding Error, The Twins, Number One, Evacuation Plan, Realignment, Familiar Feelings, Case File 01.045.mkv into Red vs. Blue - S09 [ABD: False] [ANIME: False] [whichReg: [u'season_only', u'season_only']]
2020-02-22 16:51:54 DEBUG SHOWQUEUE-REFRESH :: 78896: Creating episode object from /mnt/DS1512-Write/TV/Red vs Blue/Season 09/Red vs. Blue - S09E01E02E03E04E05E06E07E08E09E10E11E12E13E14E15E16E17E18E19E20 - Rounding Error, The Twins, Number One, Evacuation Plan, Realignment, Familiar Feelings, Case File 01.045.mkv
2020-02-22 16:51:54 DEBUG SHOWQUEUE-REFRESH :: 78896: Creating episode from Red vs. Blue - S09E01E02E03E04E05E06E07E08E09E10E11E12E13E14E15E16E17E18E19E20 - Rounding Error, The Twins, Number One, Evacuation Plan, Realignment, Familiar Feelings, Case File 01.045.mkv
```

I know that these are ridiculously long file names, but they have been working for a long time in the past but no longer are.  I suspect the length of the filename is the issue, but again...not sure what changed.

**Comment by miigotu** [comment]
We only allow like 4 episodes in one. I forget the exact number, but it is done because of silly people putting episode names and files with 30 episodes in them. Scene releases never do that, and we only support scene releases.

**Comment by ulrick65** [comment]
Are you saying I'm silly?  😁
Actually, I didn't make these and not sure where they came from like that.

I agree, it's silly though...but it's odd because I am not even sure how these got in there in the first place, but they are definitely in the database as they show up in the episode list with a mile long filename (just the wrong location).  The only reason I noticed the error is that I moved the root location and did a mass update and these all bombed out.

I don't care about these anyway...so I will delete the show at this point as they continue to throw errors and I want them out of the database or fixed (which sounds like not possible).

Thanks.
